kolibri/core/device/kolibri_plugin.py

```diff
@@ -1,11 +1,13 @@
 import time
 
 from kolibri.core.auth.hooks import FacilityDataSyncHook
+from kolibri.core.utils.lock import retry_on_db_lock
 from kolibri.plugins.hooks import register_hook
 
 
 @register_hook
 class SyncQueueStatusHook(FacilityDataSyncHook):
+    @retry_on_db_lock
```

Comment:
Just to check my understanding - these hooks are executed outside the transaction for syncs right?

Reply:
Yes that was the change we made a while back (and then re-did it)